### ISSUE TITLE
Enter does not submit in "Go to my Project" widget

### ISSUE THREAD

**Issue by sloria** [CenterForOpenScience/osf.io#2145]
## Steps

- Go to your dashboard
- Enter a project in the Go to My Project widget
- Hit Enter

## Expected

You go to the project.

## Actual

Nothing happens. :angry:

**Comment by brianjgeiger** [comment]
I expect some of this is Typeahead. Could you post a screenshot of the state of the screen after you hit enter? This may be useful when implementation happens.

**Comment by brianjgeiger** [comment]
Okay, I tested this on production, and the reason that it doesn't is just because you have the option to choose a component after. If you hit enter again after successfully choosing the project, it will take you there.

If we removed the second stage, then this *might* be something that could be done with a single enter, but I expect that Typeahead will make that difficult. Not impossible, though, and maybe not that hard. Just immediately going to the project after selecting it. But with the current UI, it would be ill-advised to do that.

**Comment by samanehsan** [comment]
This issue has been reported in #1238 and @huangginny put in a PR for it here: https://github.com/CenterForOpenScience/osf/pull/1216

**Comment by brianjgeiger** [comment]
I talked with @sloria. In Safari, which is what I first tested with, it works as intended. You hit enter to select the project in the Typeahead list, that keeps focus, then you hit enter again and you go to the project.

In Chrome, which is what Steve uses, hitting enter to confirm the Typeahead selection causes focus to be lost, so hitting enter again does not take you to the project. Ideally focus shouldn't be lost, and the form should submit.

In Firefox, hitting enter to confirm the Typeahead selection does not appear to cause focus to be lost, so hitting enter again *should* work. But it doesn't. So that's even worse than Chrome.

I have not tested on IE, but whoever takes this issue should probably do that, too.
